framework/config/config.py

```diff
@@ -183,6 +185,30 @@ def LoadTargets(self, options):
                 logging.error("%s" % e.parameter)
         return added_targets
 
+    def GetAuxTarget(self, options):
```

Comment:
Give a pythonic function name - ie. lowercase separated by '_'. I am soon going to change this throughout the codebase, so makes sense to do it here itself.